cli/script/command-parser.ts

```diff
@@ -41,10 +42,23 @@ function updateCheck(): void {
 
 function accessKeyAdd(commandName: string, yargs: yargs.Argv): void {
     isValidCommand = true;
-    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " <description>")
-        .demand(/*count*/ 3, /*max*/ 3)  // Require exactly two non-option arguments.
-        .example("access-key " + commandName + " \"VSO Integration\"", "Creates a new access key with the description \"VSO Integration\"");
+    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " <accessKeyName>")
+        .demand(/*count*/ 3, /*max*/ 3)  // Require exactly three non-option arguments.
+        .example("access-key " + commandName + " \"VSTS Integration\"", "Creates a new access key with the name \"VSO Integration\" which expires by default in 60 days")
+        .example("access-key " + commandName + " \"One time key\" --maxAge 5m", "Creates a new access key with the name \"One time key\" which expires in 5 minutes")
+        .option("maxAge", { default: null, demand: false, description: "A duration string specifying the time for which the access key remains valid for use", type: "string" });
+
+    addCommonConfiguration(yargs);
+}
 
+function accessKeyEdit(commandName: string, yargs: yargs.Argv): void {
```

Comment:
I need to think about this a little but, but my initial feeling here is that I'd prefer to have a "rename" command (like we have for app and deployment already), and then a "renew" command that specifically refreshed the TTL of the key. The use of rename feels important for consistency, and I can't help but feel like a renew command seems more intuitive than editing a maxAge field, especially since we don't display the max age value in the access key list. We show the remaining time left.